app/javascript/elm/src/SessionsList.elm

```diff
@@ -0,0 +1,188 @@
+port module SessionsList exposing (Model, Msg(..), Session, ShortType, checkedSession, init, main, subscriptions, update, updateSessions, view, viewSession, viewShortType, loadMoreSessions)
+
+import Browser
+import Html exposing (Html, button, dd, div, dl, dt, h1, img, input, label, li, section, span, text, ul)
+import Html.Attributes exposing (checked, class, for, id, src, style, type_)
+import Html.Events exposing (onClick)
+import String exposing (fromInt)
```

Comment:
It would be cool if we used the same importing/exposing style, so it will be easier to merge later. In other elm files we have:
- import Html exposing (whatever you need)
- import Html.Attributes as Attr
- import Html.Events as Events

but I'm open to using a different strategy as long as it's consistent. What do you think?

Reply:
Will use that, thanks!